src/MySqlConnector/MySqlConnection.cs

```diff
@@ -628,6 +628,8 @@ public override string ConnectionString
 		}
 	}
 
+	public string? SessionConnectionString => m_session?.ConnectionString;
```

Comment:
Is this just for tests? We shouldn't add a new public property just to make testing easier; that will imply a commitment to support this.